engines/build.info

```diff
@@ -43,7 +43,7 @@ IF[{- !$disabled{"engine"} -}]
       ENDIF
     ENDIF
     IF[{- !$disabled{"devcryptoeng"} -}]
-      ENGINES=devcrypto
+      MODULES{engine}=devcrypto
```

Comment:
Make sure to preserve this change on your next rebase.  There's a recent correct ENGINES to MODULES, but where the attribute was forgotten

Reply:
Good thing you brought that to my attention.  I was about to drop that commit. Thanks.